build/README.md

```diff
@@ -57,10 +57,37 @@ which may or may not work (and are not officially supported).
 
 ## Basic Process
 
-- Edit `build/builder/Dockerfile` as desired
-- Run `build/builder.sh init` to test -- this will build the image locally. Beware this can take a lot of time. The result of `init` is a docker image version which you can subsequently stick into the `version` variable inside the `builder.sh` script for testing locally.
-- Once you are happy with the result, run `build/builder.sh push` which pushes your image towards Docker hub, so that it becomes available to others. The result is again a version number, which you then *must* copy back into `builder.sh`. Then commit the change to both Dockerfile and `builder.sh` and submit a PR.
-- Finally, use this version number to update the `builder.dockerImage` configuration parameter in TeamCity under the [`Cockroach`](https://teamcity.cockroachdb.com/admin/editProject.html?projectId=Cockroach&tab=projectParams) and [`Internal`](https://teamcity.cockroachdb.com/admin/editProject.html?projectId=Internal&tab=projectParams) projects.
+There are two ways to build and deploy a new `builder` image; choose whichever
+is more convenient for you.
+
+### Using TeamCity to build and deploy
+
+- Edit `build/builder/Dockerfile` as desired and commit your changes.
```

Comment:
Heh. I didn't do a good enough job describing what I was thinking. How about a change like this to the first text you had put in (with only one way of pushing the image to docker hub via the TC build configuration).

```suggestion
- Edit `build/builder/Dockerfile` as desired
- Run `build/builder.sh init` to test -- this will build the image locally. Beware this can take a lot of time. The result of `init` is a docker image version which you can subsequently stick into the `version` variable inside the `builder.sh` script for testing locally.
 - Commit your changes once you are happy with the result.
```

Reply:
I think that this is more or less what you meant? Let me know.